src/test/UniswapV3SplitRouteCallee.t.sol

```diff
@@ -0,0 +1,375 @@
+        hevm.store(linkPipAddr, keccak256(abi.encode(address(this), uint256(0))), bytes32(uint256(1)));
+        linkPip.kiss(address(this));
+    }
+
+    function setEnvVars() private {
+        UniswapV3Router2 = hevm.envOr("UniswapV3Router2", 0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45);
+        uniswapTxDataNoProfit = hevm.envOr(
+            "UNISWAP_TX_DATA_NO_PROFIT",
+            hex"0000000000000000000000000000000000000000000000000000000065ba477d0000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000124b858183f00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000dead0000000000000000000000000000000000000000000000b01216b3936f60727d0000000000000000000000000000000000000000000004521f9d637a369a8de90000000000000000000000000000000000000000000000000000000000000059514910771af9ca656af840dff83e8264ecf986ca000bb8c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20001f4a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000646b175474e89094c44da98b954eedeac495271d0f0000000000000000000000000000000000000000000000000000000000000000000000"
+        );
+        uniswapTxDataProfit = hevm.envOr(
+            "UNISWAP_TX_DATA_PROFIT",
+            hex"0000000000000000000000000000000000000000000000000000000063c7bba90000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000124b858183f00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000dead0000000000000000000000000000000000000000000000eb50c6636ac5f312f00000000000000000000000000000000000000000000005a1d6fc842061b484280000000000000000000000000000000000000000000000000000000000000059514910771af9ca656af840dff83e8264ecf986ca000bb8c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20001f4a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000646b175474e89094c44da98b954eedeac495271d0f0000000000000000000000000000000000000000000000000000000000000000000000"
+        );
+
+        /* Uncomment the following lines to use historical data related to the hardcoded UNISWAP_TX_DATA_* above */
+        /* uint256 uniswapBlock = hevm.envOr("UNISWAP_BLOCK", 16432621); */
+        /* hevm.rollFork(uniswapBlock); */
+    }
+
+    function setUp() public {
+        setAddresses();
+        setInterfaces();
+        setEnvVars();
+        ali = new VaultHolder(vat);
+        aliAddr = address(ali);
+        dan = new UniswapV3SplitCallee(address(0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45), daiJoinAddr);
```

Comment:
Ideally, the router address would be defined as a constant that would be used here.

Reply:
extracted